### DISCUSSION TITLE
Incorporation idea?

### DISCUSSION THREAD

**Discussion by akthe-at** [sravioli/wezterm#5] [Ideas]
Sravioli I was curious what you thought of this guy's work? https://github.com/wez/wezterm/discussions/5435 I added his first version of this that only added/changed fonts but once he made this version I had a hard time incorporating his work with the color schemes of your config?

**Comment by sravioli** [comment]
Looks like a pretty interesting idea. Today I tried implementing a simple theme switcher but with little to no success.
I'll definitely look into this in the future.

**Comment by sravioli** [comment]
I finally implemented it! Sadly the theme list is not so long due to the fact that lots of themes do not implement a `tab_bar` colorscheme. I'll maybe fix this in the future

**Reply by akthe-at** [reply]
I will help to contribute more of the themes manually! I know I had added variations of things like Dracula/Rose Pine/etc myself to match your config setup...! :) Thanks for doing this!

**Reply by sravioli** [reply]
No problem :)
I should add that I've added a very simple `InputSelector` utility class that makes it a bit easier to implement (hopefully) any picker. It's [./picker/generic.lua](https://github.com/sravioli/wezterm/blob/main/picker/generic.lua)

**Reply by akthe-at** [reply]
if I find some time this weekend or next I will contribute some pickers for making sessions (workspaces) based on searching for git projects, fonts, I think there is another picker or two that I use but am drawing a blank on right now... Because I use a hodge-podge mixture of your config (as the main base) that is mixed with kevin's, and a bunch of random things here and there that I have picked up over time or made myself it will take me a few days to adapt all of your changes you implemented over the last week since I can't do a nice easy git rebase/merge etc

**Reply by sravioli** [reply]
No worries! I appreciate the fact that you're willing to work on someone else's config
